### PR TITLE
Revert "Typos in syndicate uplink descriptions"

### PR DESCRIPTION
**Pull request by optimumtact**
Reverts tgstation/tgstation#27878

for violation of contribution guidelines

### CODE REVIEW

**Comment by Cheridan** [comment]
🎆

**Comment by optimumtact** [comment]
YOU ARE DEAD TO ME CHERIDAN

DEAD

File: code/modules/uplink/uplink_item.dm [hunk 1]
```diff
@@ -1121,14 +1121,14 @@ GLOBAL_LIST_EMPTY(uplink_items) // Global list so we only initialize this once.
 
 /datum/uplink_item/device_tools/codespeak_manual
 	name = "Codespeak Manual"
-	desc = "Syndicate agents can be trained to use a series of codewords to convey complex information, which sounds like random concepts and drinks to anyone listening. This manual teaches you this Codespeak. You can also hit someone else with the manual in order to teach them. One use."
+	desc = "Syndicate agents can be trained to use a series of codewords to comvey complex information, which sounds like random concepts and drinks to anyone listening. This manual teaches you this Codespeak. You can also hit someone else with the manual in order to teach them. One use."
```
Comment (ChangelingRain) [inline code/modules/uplink/uplink_item.dm]: actual typo

File: code/modules/uplink/uplink_item.dm [hunk 2]
```diff
@@ -1121,14 +1121,14 @@ GLOBAL_LIST_EMPTY(uplink_items) // Global list so we only initialize this once.
...
 	item = /obj/item/weapon/codespeak_manual
 	cost = 2
 	exclude_modes = list(/datum/game_mode/nuclear)
 
 /datum/uplink_item/device_tools/codespeak_manual_deluxe
 	name = "Deluxe Codespeak Manual"
-	desc = "Syndicate agents can be trained to use a series of codewords to convey complex information, which sounds like random concepts and drinks to anyone listening. This manual teaches you this Codespeak. You can also hit someone else with the manual in order to teach them. This is the deluxe edition, which has unlimited uses."
+	desc = "Syndicate agents can be trained to use a series of codewords to comvey complex information, which sounds like random concepts and drinks to anyone listening. This manual teaches you this Codespeak. You can also hit someone else with the manual in order to teach them. This is the deluxe edition, which has unlimited uses."
```
Comment (ChangelingRain) [inline code/modules/uplink/uplink_item.dm]: actual typo

File: code/modules/uplink/uplink_item.dm [hunk 3]
```diff
@@ -1247,7 +1247,7 @@ GLOBAL_LIST_EMPTY(uplink_items) // Global list so we only initialize this once.
 
 /datum/uplink_item/role_restricted/mimery
 	name = "Guide to Advanced Mimery Series"
-	desc = "The classical two part series on how to further hone your mime skills. Upon studying the series, the user should be able to make 3x1 invisible walls, and shoot bullets out of their fingers. Obviously only works for Mimes."
+	desc = "The classical two part series on how to further hone your mime skills. Upon studying the series, the user should be able to make 3x1 invisble walls, and shoot bullets out of their fingers. Obviously only works for Mimes."
```
Comment (ChangelingRain) [inline code/modules/uplink/uplink_item.dm]: actual typo

File: code/modules/uplink/uplink_item.dm [hunk 4]
```diff
@@ -1359,7 +1359,7 @@ GLOBAL_LIST_EMPTY(uplink_items) // Global list so we only initialize this once.
 
 /datum/uplink_item/badass/surplus
 	name = "Syndicate Surplus Crate"
-	desc = "A dusty crate from the back of the Syndicate warehouse. Rumored to contain a valuable assortment of items, \
+	desc = "A dusty crate from the back of the Syndicate warehouse. Rumored to contain a valuable assortion of items, \
```
Comment (ChangelingRain) [inline code/modules/uplink/uplink_item.dm]: assortion isn't a word even if it looks like one

File: code/modules/uplink/uplink_item.dm [hunk 5]
```diff
@@ -1052,7 +1052,7 @@ GLOBAL_LIST_EMPTY(uplink_items) // Global list so we only initialize this once.
 /datum/uplink_item/device_tools/rad_laser
 	name = "Radioactive Microlaser"
 	desc = "A radioactive microlaser disguised as a standard Nanotrasen health analyzer. When used, it emits a \
-			powerful burst of radiation, which, after a short delay, can incapacitate all but the most protected \
+			powerful burst of radiation, which, after a short delay, can incapitate all but the most protected \
```
Comment (ChangelingRain) [inline code/modules/uplink/uplink_item.dm]: incapitate; to reattach a head

File: code/modules/uplink/uplink_item.dm [hunk 6]
```diff
@@ -1014,7 +1014,7 @@ GLOBAL_LIST_EMPTY(uplink_items) // Global list so we only initialize this once.
 /datum/uplink_item/device_tools/powersink
 	name = "Power Sink"
 	desc = "When screwed to wiring attached to a power grid and activated, this large device places excessive \
-			load on the grid, causing a station-wide blackout. The sink is large and cannot be stored in most \
+			load on the grid, causing a stationwide blackout. The sink is large and cannot be stored in most \
```
Comment (ChangelingRain) [inline code/modules/uplink/uplink_item.dm]: give me back my hyphen

File: code/modules/uplink/uplink_item.dm [hunk 7]
```diff
@@ -898,7 +898,7 @@ GLOBAL_LIST_EMPTY(uplink_items) // Global list so we only initialize this once.
 
 /datum/uplink_item/device_tools/military_belt
 	name = "Chest Rig"
-	desc = "A robust seven-slot set of webbing that is capable of holding all manner of tactical equipment."
+	desc = "A robust seven-slot set of webbing that is capable of holding all manner of tatical equipment."
```
Comment (ChangelingRain) [inline code/modules/uplink/uplink_item.dm]: typo

**Comment by kevinz000** [comment]
lmao

**Comment by PKPenguin321** [comment]
Let go

File: code/modules/uplink/uplink_item.dm [hunk 8]
```diff
@@ -203,7 +203,7 @@ GLOBAL_LIST_EMPTY(uplink_items) // Global list so we only initialize this once.
 /datum/uplink_item/dangerous/shotgun
 	name = "Bulldog Shotgun"
 	desc = "A fully-loaded semi-automatic drum-fed shotgun. Compatiable with all 12g rounds. Designed for close \
-			quarter anti-personnel engagements."
```
Comment (Shadowlight213) [inline code/modules/uplink/uplink_item.dm]: Actual typo

**Comment by BeeSting12** [comment]
what guideline was broken anyways seems like legit grammar fixes to me

**Comment by RemieRichards** [comment]
@BeeSting12 what was a normal typo fix PR changed when the author decided to make things """"consistent"""" by changing the spellings of certain words to match the most common form (Armor was used 6 times, where Armour only used 3 so they made them all Armory)

We've had many fights over this before, usually started by the american, and even a PR that replaced all british language in the game with american ones by firecage (if it wasn't all, it was a LOT of it).

No consensus could be agreed because people from commonwealth nations write in British English, and so would always write that way, and the inverse is true for the Americans and American English.

In the end we simply decided that it is something you just don't change, it's up to the developer to write it how they please in each case, and not to change what is already written.

**Comment by Cyberboss** [comment]
I mean, sure that's fine, but I would definitely prioritize in-game consistency. Having something named one way, and then having it mentioned somewhere else another way is bad. I don't care which spelling we use as long as the stuff that uses it is consistent.

**Comment by Steelpoint** [comment]
Lets livestream someone flipping a coin to decide if we use armor or armour.

**Comment by Bawhoppen** [comment]
@Steelpoint People get really invested in this so they'd find a way to rig it

**Comment by Bawhoppen** [comment]
you just selfmerged

**Pull request merged**